src/identity/components/GlobalHeader/AccountDropdown.tsx

```diff
@@ -50,6 +50,11 @@ export const AccountDropdown: FC<Props> = ({
       iconFont: IconFont.CogSolid_New,
       href: `/orgs/${activeOrg.id}/accounts/settings`,
     },
+    {
+      name: 'Organizations',
+      iconFont: IconFont.Group,
+      href: `/orgs/{activeOrg.id}/accounts/orgslist`,
+    },
```

Comment:
![Screen Shot 2022-10-28 at 4 35 38 PM](https://user-images.githubusercontent.com/91283923/198739133-c8537fbc-476e-4664-9c4f-3c51b837c32f.png)
This icon will need to change to this - an icon that hasn't been added to clockface yet. Julia is good with using the Group icon in the interim, until the clockface change is out.